spec/cli_v1_commands.yaml

```diff
@@ -1,7 +1,7 @@
 clispec: '001'
 info:
   id: cli
-  version: 1.82.2
+  version: latest
```

Comment:
@sweatybridge is this okay? Or should we wait for the new release and then generate the reference docs with the right semver

Reply:
We can just keep the current semver since only docs are updated.